.gitignore

```diff
@@ -4,3 +4,4 @@ vendor
 build
 assets/dist
 /*.zip
+.idea
```

Comment:
Maybe should it be part of another PR?

Reply:
Yeah I sneaked that in because IDE files were showing up. It's already in the main Sensei repo - we can sync this file too at some point.

Reply:
Ahh, I probably accidentally removed it as I copied and pasted this file across all repos. 

I have `.idea` in my global gitignores file.